core/src/main/scala/scalaz/std/String.scala

```diff
@@ -38,47 +38,32 @@ trait StringFunctions {
 
   // Parsing functions.
 
-  def parseBoolean(s:String): Validation[IllegalArgumentException, Boolean] = try {
-    Success(s.toBoolean)
-  } catch {
-    case e: IllegalArgumentException => Failure(e)
-  }
+  def parseBoolean(s:String): Validation[IllegalArgumentException, Boolean] =
+    \/.fromTryCatchThrowable[Boolean, IllegalArgumentException](s.toBoolean).validation
```

Comment:
There is `fromTryCatchThrowable` in Validation as well.

```scala
Validation.fromTryCatchThrowable[Boolean, IllegalArgumentException](s.toBoolean)
```

https://github.com/scalaz/scalaz/blob/v7.3.0-M2/core/src/main/scala/scalaz/Validation.scala#L390